packages/common/src/hooks/useAudioBalance.ts

```diff
@@ -27,6 +27,6 @@ export const useTotalBalanceWithFallback = () => {
       return new BN(account.total_balance) as BNWei
     }
 
-    return null
+    return new BN(0) as BNWei
```

Comment:
reason for this change?

Reply:
typecheck was failing when this fn returned `null`. and the name of the fn has `withFallback` so i think we should fallback to 0.